Documentation/conf.py

```diff
@@ -270,6 +270,31 @@
 # Try as hard as possible to find references
 default_role = 'any'
 
+# -- Options for Link Check output ----------------------------------------
+
+linkcheck_ignore = [
+    # Local links
+    'http://127.0.0.1',
+    'http://localhost',
+    'http://192.168.60.13',
+    'https://192.168.60.11',
+
+    # URLs that are not reachable(invalid or test links)
+    'https://bookinfo.cilium.rocks',
+    'dns:query',
+
+    # Needs authentication
+    'https://github.com/cilium/cilium/projects/new',
+    'https://github.com/cilium/cilium/settings/branches',
+    'https://jenkins.cilium.io/job/',
+    'https://jenkins.cilium.io/view/'
+]
+
+linkcheck_anchors = False
+linkcheck_retries = 5
+
+tls_verify=False
```

Comment:
```suggestion
tls_verify = False
```

Do you happen to know whether this option is used for anything else than the linkchecker? It's fine to disable for the linkchecker, but I wouldn't want to lower security if it were used for something else :thinking:

Reply:
https://github.com/sphinx-doc/sphinx/pull/3156/files#diff-aac07aefec754086f5c344e2c233862f6a611c739b5647fd456c1b77d3577c79R10-R12, it is used for `intersphinx` as well

Reply:
They should really document that. Thanks a lot for looking it up! OK, intersphinx is not a concern and I don't see us using it anytime soon, so all good :+1:.